workers.py

```diff
@@ -9,11 +9,19 @@
 import prometheus_metrics
 
 import custom_parser
+import target_worker
 
 logger = logging.getLogger()
 CHUNK = 10240
 MAX_RETRIES = 3
 
+ENTITIES = [
```

Comment:
This should be abstracted to `target_worker/__init__.py` as additional data for the topological worker. No need to have it global for all use cases.. But, it can be left here for now. I'll move it when I'll do a code shuffle so both workers are cleaner...

Reply:
Yep, no need for this global.

Moved this to the init file as well.
The `target_worker` library would now provide all the necessary raw materials for each worker to do it's thing.